<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>878</width>
                <height>480</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>878</width>
                <height>480</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>878</width>
                <height>480</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="font">
                                    <font>
                                        <family>Rockwell</family>
                                        <pointsize>20</pointsize>
                                        <bold>true</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>File Compression and Decompression</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QFormLayout" name="formLayout">
                                <item row="0" column="0">
                                    <widget class="QLineEdit" name="ChooseFile">
                                        <property name="minimumSize">
                                            <size>
                                                <width>700</width>
                                                <height>25</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>16777215</width>
                                                <height>16777215</height>
                                            </size>
                                        </property>
                                        <property name="readOnly">
                                            <bool>false</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QPushButton" name="BtnFileChoose">
                                        <property name="maximumSize">
                                            <size>
                                                <width>200</width>
                                                <height>16777215</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Choose File</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QRadioButton" name="RdZip">
                                        <property name="text">
                                            <string>Zip Compression</string>
                                        </property>
                                        <property name="checked">
                                            <bool>true</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QRadioButton" name="RdRAR">
                                        <property name="text">
                                            <string>TAR Compression</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QRadioButton" name="RdBzip">
                                        <property name="text">
                                            <string>BZip Compression</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QRadioButton" name="Rd7z">
                                        <property name="text">
                                            <string>7z Compression</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <item>
                                    <widget class="QPushButton" name="BtnCompress">
                                        <property name="text">
                                            <string>Compress</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="BtnDecompress">
                                        <property name="text">
                                            <string>Decompress</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="BtnMainMenu">
                                        <property name="text">
                                            <string>Close</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_5">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <spacer name="verticalSpacer_4">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
